### PR TITLE
✨ Add Card Grid components

### PR DESCRIPTION
**Pull request by fre-ben**
- Add Player Card Grid
- Add Opponent Card Grid
- Style Card Grids
- Add to storybook

Deployment: https://sisu-pipelin-cardgrid-ndciozcz.herokuapp.com/storybook/index.html?path=/story/common-gameboard--cardgrid
Part of issue #6

### CODE REVIEW

**Review by DNaunin — APPROVED** [review]
LGTM! 🎉

File: components/gameboard/CardGrid.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      <section className={styles.container}>
+        <p>
+          Score: <span>{players[0].score}</span>
+        </p>
+        <p>{players[0].name}</p>
+        <div className={styles.lines}>
+          <div className={styles.cardGrid}>{createPlayerCardGrid}</div>
+        </div>
+      </section>
+    </>
```
Comment (mhstrkmp) [inline components/gameboard/CardGrid.tsx]:
```suggestion
      <section className={styles.container}>
        <p>
          Score: <span>{players[0].score}</span>
        </p>
        <p>{players[0].name}</p>
        <div className={styles.lines}>
          <div className={styles.cardGrid}>{createPlayerCardGrid}</div>
        </div>
      </section>
```
I think you won't need the fragments here as everything is wrapped in the `<section>`

File: components/gameboard/OpponentCardGrid.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  return (
+    <>
+      <section className={styles.opponentContainer}>
+        <p>
+          Score: <span>{players[0].score}</span>
+        </p>
+        <p>{players[0].name}</p>
+        <div className={styles.oppCardGrid}>{createPlayerCardGrid}</div>
+      </section>
+    </>
```
Comment (mhstrkmp) [inline components/gameboard/OpponentCardGrid.tsx]:
```suggestion
      <section className={styles.opponentContainer}>
        <p>
          Score: <span>{players[0].score}</span>
        </p>
        <p>{players[0].name}</p>
        <div className={styles.oppCardGrid}>{createPlayerCardGrid}</div>
      </section>
```
same as above

**Review by mhstrkmp — COMMENTED** [review]
good work 👍

File: components/gameboard/CardGrid.tsx (hunk 1 above)
Comment (fre-ben) [reply components/gameboard/CardGrid.tsx]: Yup, will remove them - thanks!

**Pull request merged**